src/submission/submission-to-clinical/stat-calculator.ts

```diff
@@ -105,26 +109,31 @@ const getEmptyCoreStats = (): CompletionStats => ({
  * The individual core entities will still be calculated so the current list of complete entities for this donor will still be known.
  *
  * @param donor
- * @param hasMissingEntityException - Indicates if this donor has been given an exception to allow it to be core complete while missing select core entities
+ * @param exceptions
+ * @param exceptions.hasMissingEntityException - Indicates if this donor has been given an exception to allow it to be core complete while missing select core entities
+ * @param exceptions.hasSingleSpecimenException - Indicates if there is an exception to allow just the tumour specimen sample
  * @returns
  */
 export const calculateDonorCoreCompletionStats = (
 	donor: Donor,
-	hasMissingEntityException: boolean,
+	exceptions: { hasMissingEntityException: boolean; hasSingleSpecimenException: boolean },
 ): CompletionStats => {
 	const updatedCompletionStats: CompletionStats = donor.completionStats
 		? cloneDeep(donor.completionStats)
 		: getEmptyCoreStats();
 
-	updatedCompletionStats.hasMissingEntityException = hasMissingEntityException;
+	updatedCompletionStats.hasMissingEntityException = exceptions?.hasMissingEntityException;
```

Comment:
Pprobably don't need the `?.` here since the exceptions object is required, but no harm.